server/test-api-curl.sh

```
@@ -0,0 +1,381 @@
+run_test "Test Invalid Authentication" test_invalid_auth
+run_test "Test Validation Errors" test_validation_errors
+run_test "Test Rate Limiting" test_rate_limiting
+
+# Print results
+echo -e "\n${BLUE}📊 Test Results Summary${NC}"
+echo "========================="
+echo -e "${GREEN}✅ Passed: $PASSED${NC}"
+echo -e "${RED}❌ Failed: $FAILED${NC}"
+
+if [ $((PASSED + FAILED)) -gt 0 ]; then
+    local success_rate=$((PASSED * 100 / (PASSED + FAILED)))
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)